specification/sections/02-providers.md

```diff
@@ -168,7 +168,7 @@ class MyProvider implements Provider {
 
 Many feature flag frameworks or SDKs require some initialization before they can be used.
 They might require the completion of an HTTP request, establishing persistent connections, or starting timers or worker threads.
-The `initialization` function is an ideal place for such logic.
+The `initialize` function is an ideal place for such logic. The `initialize` function may be renamed in accordance with language idioms.
```

Comment:
```suggestion
The `initialize` function is an ideal place for such logic.
The `initialize` function is named in accordance with language idioms.
```

I try to avoid RFC2119 terms in non-normative text, and speak in present tense, but this is a nit.